lti_consumer/utils.py

```diff
@@ -58,3 +58,26 @@ def get_lms_lti_access_token_link(location):
         lms_base=get_lms_base(),
         location=text_type(location),
     )
+
+
+def get_lms_ags_endpoint(location, pk=None, action=""):
```

Comment:
@davidjoy @davestgermain This is basically replicating a DRF viewset with the readonly views here, and will be replaced by an actual ModelViewSet once the full LineItem Implementation is done.